typegen/unification/drop_min_threshold.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+
+from .filter_base import TraceDataFilter
+
+import constants
+
+
+class MinThresholdFilter(TraceDataFilter):
+    """Drops all rows whose types appear less often than the minimum threshold."""
+
+    ident = "drop_min_threshold"
+
+    min_threshold: float = 0.25
+
+    def apply(self, trace_data: pd.DataFrame) -> pd.DataFrame:
+        """
+        Drops all rows whose types appear less often than the minimum threshold in the provided trace data and returns the processed trace data.
+
+        @param trace_data The provided trace data to process.
+        """
+
+        COUNT_COLUMN = "count"
```

Comment:
these should probably be static attributes on the class

Reply:
Done